src/test/java/info/wzieleziecki/WordChainTest.java

```diff
@@ -7,38 +9,38 @@
 //tests defined based on the challenge description - http://codekata.com/kata/kata19-word-chains/
 public class WordChainTest {
 
-    private DictionaryReader dictionaryReader = new DictionaryReader();
+    private Word word = new Word();
 
     @Test
-    public void test1PositiveSolution() {
+    public void solutionForRubyToCode() {
```

Comment:
czemu nazwa solution zamiast test? Nie jestem w testach za dobry ale o ile się orientuję to nazwa metody powinna mówić o tym co ona testuje. Słowo solution nic w tym miejscu mi nie wyjaśnia

Reply:
Zmieniłem nazwy metod zgodnie z uwagą mmielko